lib/migrations/2_deploy_organization.ts

```diff
@@ -29,12 +30,12 @@ export const arcJsDeployer = (web3: Web3, artifacts: any, deployer: any): void =
   // so Utils.getWeb3 can find it
   (global as any).web3 = web3;
 
-  const network = ConfigService.get("network") || "ganache";
+  const network = env.arcjs_network || "ganache";
```

Comment:
why this change?

Reply:
I can't think of a use case for enabling the migration-related env variables to be defined in the config system.  It adds needless complication.  So I removed it.  Let me know if you can think of a use case.